_data/reviews.yml

```
@@ -39,6 +39,13 @@
       feedback: Sonali gave me really good advice about how to approach being in a new backend team that wasn't turning out exactly as I had had expected, and how to go about job hunting when I am in this transitional state of moving from front end to backend. 
       name: Anonymous
       type: Ad-Hoc
+    - score: 5
+      feedback: |
+        Definitely yes, I came to the session with lot of questions and indirection. But, Sonali was able to give me right suggestions and resources to navigate and move forward.
```

This can be edited a but at the start to make it easier presented as feedback

```suggestion
        I came to the session with lot of questions and indirection. But, Sonali was able to give me right suggestions and resources to navigate and move forward.
```

---

fixed